riva/clients/tts/riva_tts_client.cc

```diff
@@ -42,6 +43,16 @@ DEFINE_bool(
     "Whether to use SSL credentials or not. If ssl_cert is specified, "
     "this is assumed to be true");
 DEFINE_string(metadata, "", "Comma separated key-value pair(s) of metadata to be sent to server");
+DEFINE_string(
+    zero_shot_audio_prompt, "",
+    "Input audio file for Zero Shot Model. Audio length between 0-3 seconds.");
+DEFINE_string(
+    zero_shot_encoding, "", "Input audio encoding(PCM/OPUS) of audio file for Zero Shot Model.");
+DEFINE_int32(zero_shot_sample_rate, 22050, "Sample rate of input prompt.");
```

Comment:
should this be empty ? as the sample rate would come from file data.

Reply:
I'm thinking of removing these altogether. They can be picked from file itself, instead of asking user to input them explicitly.